presto-main/src/main/java/com/facebook/presto/server/security/KerberosConfig.java

```diff
@@ -47,6 +51,7 @@ public KerberosConfig setServiceName(String serviceName)
         return this;
     }
 
+    @NotNull
```

Comment:
Removed that check because we need to have the keytab if Kerberos is being used for authentication for CLI